zuul.d/whitebox_neutron_tempest_jobs.yaml

```diff
@@ -0,0 +1,149 @@
+              ^whitebox_neutron_tempest_plugin.*reboot_node
+              ^whitebox_neutron_tempest_plugin.*test_previously_used_ip
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_internal_dns.InternalDNSInterruptions.*
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_l3ha_ovn.*
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_metadata_rate_limiting
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_ovn_dbs.OvnDbsMonitoringTest.*
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_qos.*external
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_security_group_logging
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_extra_dhcp_opts.ExtraDhcpOptionsTest.test_extra_dhcp_opts_ipv4_ipv6_stateless
+            # https://review.opendev.org/892839
+            # - neutron_tempest_plugin.scenario.test_mtu.NetworkWritableMtuTest
+            # It's in Blacklist before, FWaaS tests are not executed in any of our setups so there is no need to keep them whitelisted
+            #   ^neutron_tempest_plugin.fwaas.*
+            # Note(chandankumar): Skipping due to https://issues.redhat.com/browse/OSPRH-9091/RHOSZUUL-1940 
+            # and It require xl node to fix no valid host.
+            # - whitebox_neutron_tempest_plugin.tests.scenario.test_extra_dhcp_opts.ExtraDhcpOptionsTest.test_extra_dhcp_opts_ipv4_ipv6_stateless
+        - stepName: single-thread-testing
+          tempestRun:
+            concurrency: 1
+            includeList: |
+              whitebox_neutron_tempest_plugin.*south_north
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_security_group_logging
+              test_multicast.ext
+              test_multicast.*restart
+              whitebox_neutron_tempest_plugin.*south_north
+              whitebox_neutron_tempest_plugin.tests.scenario.*test_mtu
+              ^neutron_.*plugin..*scenario.test_.*macvtap
+              ^whitebox_neutron_tempest_plugin.many_vms
+              ^whitebox_neutron_tempest_plugin.ovn_controller_restart
+              ^whitebox_neutron_tempest_plugin.test_previously_used_ip
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_internal_dns.InternalDNSInterruptions.
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_l3ha_ovn.
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_metadata_rate_limiting
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_ovn_dbs.OvnDbsMonitoringTest.
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_qos.*external
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_router_flavors
+              ^whitebox_neutron_tempest_plugin.tests.scenario.test_vlan_transparency.ProviderNetworkVlanTransparencyTest
+              whitebox_neutron_tempest_plugin.tests.scenario.test_security_group_logging
```

Comment:
Thank you for the changes.
I noticed a small duplicate line in IncludeList for test_security_group_logging.